posthog/migrations/0257_make_tiles_relationship_looser.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.14 on 2022-08-18 17:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0256_add_async_deletion_model"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="dashboard", name="insights",),
```

Comment:
hmmm maybe this isn't safe for cloud

Might need a release that adds the properties with different names. And then a release that removes the use of the relationship, And finally one that removes it and renames the properties back to the desired name.

A faff for cloud but all in one for self-hosted

Reply:
`RemoveField` 🤔 
The rule of "no changes to db structure, sadly" still applies :(.

Reply:
Yeah, I have a version that doesn't remove the field. Still makes me sad we can't remove fields :)